### DISCUSSION TITLE
External app, central user management with Pimcore - good idea?

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#10231] [General]
Hello,

I'm new in Pimcore space and have the following situation where I would need some helping pointers :)

A client uses contently Pimcore including the web shop. Now a new web application should get developed, so far so easy. But the current existing Pimcore users should be able to use the new app by using their credentials. So I'm looking for an API to login existing users, registering new users and do actions in their context.
In the end, one central user repository should exist to avoid the need for multiple registrations for a user.

Is https://pimcore.com/docs/pimcore/current/Development_Documentation/Best_Practice/Building_Custom_Rest_APIs.html the correct starting point? Or is their a plugin for authentication to external systems already somewhere?

Thank you very much for your help!

**Comment by Nicolo-R** [comment]
I join the question!

My situation is as follows. I would like to combine PimCore with the CRM system, but so that Users do not register again, but use the accounts that they already have.
